homeassistant/components/deconz/binary_sensor.py

```diff
@@ -13,25 +15,18 @@
 ATTR_VIBRATIONSTRENGTH = 'vibrationstrength'
 
 
-async def async_setup_platform(
```

Comment:
We shouldn't remove this as it guards against a stack trace.

Reply:
I will revert this

Reply:
I have reverted it, but tests calling the specific deconz platforms didn't fail and no stack trace